### ISSUE TITLE
NVDA is announcing as 'Link slash'. It is not announcing  about the Live component

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#13708]
<img width="670" alt="image" src="https://user-images.githubusercontent.com/93735775/169155974-be9df685-c76e-4c93-a041-06aa688e02d5.png">

### Steps to reproduce:
**Pre-requisites:**
1. Launch Outlook application and sign in with valid credentials.

**Repro steps:**
1. Navigate to Mail tab.
2. Navigate to 'New message' button and activate it.
3. Navigate to 'ribbon pane' and activate 'Live component' button.
4. Navigate to any live component menu list item and activate any one component.
5. Now navigate to the Live component and listen to the NVDA behavior.

### Actual behavior:
Upon navigating through the 'Live component' in the body of the message, NVDA is announcing as 'Link slash'. It is not announcing about the Live component.

### Expected behavior:
Upon navigating through the 'Live component' in the body of the message, NVDA should announce as 'Link slash' instead it should announce about the live component. For example: 'A Live component Addin inline with text'.

### System configuration
Windows OS Version: Version 21H2 (OS Build 22000.675)
Outlook Version: 2205 (Build 15224.20168)
NVDA Version: 2021.3.5
AT Used: NVDA

#### NVDA installed/portable/running from source:
installed

#### NVDA version:
NVDA Version: 2021.3.5

#### Windows version:
Windows OS Version: Version 21H2 (OS Build 22000.675)

#### Name and version of other software in use when reproducing the issue:
Outlook Version: 2205 (Build 15224.20168)

**Comment by seanbudd** [comment]
Has this been reported to Microsoft Outlook team?

**Comment by msftedad** [comment]
fixed this issue on latest NVDA build and closing this issue.
